### ISSUE TITLE
Error in yahoo strategy

### ISSUE THREAD

**Issue by BeejLuig** [kerrykimrusso/target-blank#41]
Error and stack trace:
```
Uncaught TypeError: utils.hasSameDomain is not a function
    at Object.shouldTreatAsRelative (yahoo.js:7)
    at Object.determineAnchorType (utils.js:48)
    at HTMLAnchorElement.onAnchorClicked (contentscript.js:24)
shouldTreatAsRelative @ yahoo.js:7
determineAnchorType @ utils.js:48
onAnchorClicked @ contentscript.js:24
```

**Comment by kerrykimrusso** [comment]
fixed by #42

**Comment by BeejLuig** [comment]
yay 🙌
